app/serializers/disability_claim_base_serializer.rb

```diff
@@ -22,11 +22,11 @@ def phase_change_date
   end
 
   def open
-    object.data['claimCompleteDate'].blank?
+    object_data['claimCompleteDate'].blank?
```

Comment:
Python has an exception like `raise NotImplemented` that you put in base classes to indicate that the sub-class needs to override it to work. I don't know if there's a Ruby equivalent.